.gitignore

```diff
@@ -13,3 +13,5 @@ debug.log
 .coverage
 htmlcov
 .benchmarks
+
+.DS_Store
```

Comment:
Please remove, this is not specific to GEF (better to add it to your ~/.gitignore_global instead)

Reply:
removed